cli/src/test/scala/com/codacy/analysis/cli/MainImplSpec.scala

```diff
@@ -1,44 +0,0 @@
-package com.codacy.analysis.cli
```

Comment:
Why delete this test?

Reply:
It was flaky and this stuff should (and will) be tested by the integration tests